egs/jsut/asr1/conf/fbank.conf

```diff
@@ -0,0 +1,2 @@
+--sample-frequency=48000 
```

Comment:
We usually downsample the data to 16000 for ASR. Many front-end configurations are based on  (optimized by)16000. Could you compare 16000 vs. 48000?

You can just add the sox downsample command in `wav.scp`.
See https://github.com/espnet/espnet/blob/f7e3dbb8f1e011c41973c4c77403edf8ac684126/egs/hkust/asr1/run.sh#L107
(This example is upsample, but your case is downsample).

Reply:
OK!